src/app/features/cases/components/CaseTimeline/Events/ReasonData.tsx

```diff
@@ -2,13 +2,19 @@ import React from "react"
 import { Dl, DLProps } from "../helpers/Helpers"
 import { reasonLabelsMap } from "../helpers/dictionaries"
 
+const fields = ["start_date", "author", "reason", "description"]
+
 const ReasonData: React.FC<DLProps> = ({ thread }) => (
-    <Dl>
-      <div>
-        <dt>{ reasonLabelsMap.reason }</dt>
-        <dd>{ thread.event_values.reason }</dd>
-      </div>
-    </Dl>
-  )
+  <>
+    { fields.map(field => (
+      <Dl>
```

Comment:
Shouldn't this map return the div inside the <DL> instead of a complete <DL>?

Reply:
Yes, will fix